### PR TITLE
DOC-9707 Docs for Supportability and Reducing L2 Toil

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-9707

In show-sessions.md, updated Response field list.

### CODE REVIEW

**Review by nkodali — APPROVED** [review]
lgtm

File: src/current/v23.1/show-sessions.md [hunk 1]
```diff
@@ -37,14 +37,15 @@ Field | Description
...
+`status` | The session's status.
 `user_name` | The username of the connected user.
 `client_address` | The address and port of the connected client.
 `application_name` | The [application name]({% link {{ page.version.version }}/set-vars.md %}#supported-variables) specified by the client, if any. For sessions from the [built-in SQL client]({% link {{ page.version.version }}/cockroach-sql.md %}), this will be `cockroach`.
 `active_queries` | The SQL queries currently active in the session.
 `last_active_query` | The most recently completed SQL query in the session.
 `session_start` | The [`timestamptz`]({% link {{ page.version.version }}/timestamp.md %}) at which the session started.
-`oldest_query_start` | The [`timestamptz`]({% link {{ page.version.version }}/timestamp.md %}) at which the oldest currently active SQL query in the session started.
-`kv_txn` | The ID of the current key-value transaction for the session.
+`active_query_start` | The time when the current active query in the session was started.
```
[the remark this answers is not in the document]
Comment (florence-crl) [reply src/current/v23.1/show-sessions.md:47]: updated

File: src/current/v23.1/show-sessions.md [hunk 2]
```diff
@@ -37,14 +37,15 @@ Field | Description
...
+`num_txns_executed` | Count of the number of [transactions]({% link {{ page.version.version }}/transactions.md %}) that have been opened in this session. This count includes transactions that are in progress.
```
[the remark this answers is not in the document]
Comment (florence-crl) [reply src/current/v23.1/show-sessions.md:48]: updated

File: src/current/v24.1/show-sessions.md [hunk 3]
```diff
@@ -37,14 +37,17 @@ Field | Description
...
 `active_queries` | The SQL queries currently active in the session.
 `last_active_query` | The most recently completed SQL query in the session.
 `session_start` | The [`timestamptz`]({% link {{ page.version.version }}/timestamp.md %}) at which the session started.
-`oldest_query_start` | The [`timestamptz`]({% link {{ page.version.version }}/timestamp.md %}) at which the oldest currently active SQL query in the session started.
-`kv_txn` | The ID of the current key-value transaction for the session.
+`active_query_start` | The time when the current active query in the session was started.
+`num_txns_executed` | Count of the number of [transactions]({% link {{ page.version.version }}/transactions.md %}) that have been opened in this session. This count includes transactions that are in progress.
+`trace_id` | The ID of the session's active trace. It will be 0 if [tracing]({% link {{ page.version.version }}/set-vars.md %}#set-tracing) is off.
```
[the remark this answers is not in the document]
Comment (florence-crl) [reply src/current/v24.1/show-sessions.md:49]: updated

**Review by florence-crl — COMMENTED** [review]
TFTRs!

**Pull request merged**